.gitignore

```diff
@@ -117,3 +117,7 @@ Cargo.lock
 
 # Rethnet remote node cache
 remote_node_cache/
+
+# Brew package manager. 
+# Lock file is not recommended to be commited: https://github.com/Homebrew/homebrew-bundle#versions-and-lockfiles
+Brewfile.lock.json
```

Comment:
git show this error when a EOL (end of line) is not present at the end of the file. To avoid it, I always set my IDE to automatically end a newline at the end of my file on save.

Reply:
Thanks, fixed in e0eb4862fddce9b89c7dbd5d5d234405d9ae578e